### ISSUE TITLE
gopherwasm error undefined: js.TypedArray

### ISSUE THREAD

**Issue by benc-uk** [hajimehoshi/ebiten#950]
When trying to build as WASM, I get an error about gopherwasm 
Going to the https://github.com/gopherjs/gopherwasm repo I see it's deprecated 5 months ago and has been replaced with https://golang.org/pkg/syscall/js/

I am unable to compile my game as WASM

```
GOOS=js GOARCH=wasm go build -o gofract-mandelbrot.wasm . 
# github.com/gopherjs/gopherwasm/js
/home/ben/go/pkg/mod/github.com/gopherjs/gopherwasm@v1.1.0/js/js_wasm.go:46:19: undefined: js.TypedArray
/home/ben/go/pkg/mod/github.com/gopherjs/gopherwasm@v1.1.0/js/js_wasm.go:49:9: undefined: js.TypedArrayOf
```

**Comment by benc-uk** [comment]
Ahh I've spotted this has been fixed after 1.9.3
I ran `go get github.com/hajimehoshi/ebiten@master` and now I can build WASM

**Comment by hajimehoshi** [comment]
Right, as `syscall/js` API has been changed without compatibility, now 1.9.3 cannot be built as Wasm with Go 1.13. Use Go 1.12, or use the master of Ebiten.
